ffi/npm/Makefile

```diff
@@ -18,7 +18,7 @@ lib/index.mjs: FORCE
 ../../target/wasm32-unknown-unknown/release/icu_capi.wasm: FORCE
 	rustup toolchain install ${PINNED_CI_NIGHTLY}
 	rustup component add rust-src --toolchain ${PINNED_CI_NIGHTLY}
-	RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-args=-zstack-size=${WASM_STACK_SIZE} -Clto -Cembed-bitcode" \
+	RUSTFLAGS="-Cpanic=abort -Copt-level=s -C link-args=-zstack-size=${WASM_STACK_SIZE} -Clto -Cembed-bitcode -Zwasm-c-abi=spec" \
```

Comment:
this is going to break the demo which calls this Makefile when it installs the package but currently uses nightly

Reply:
The nightly flag is going to return soon

Reply:
How should we fix this? Should we do some toolchain detection? Or pin a nightly?